### ISSUE TITLE
Article & Article List – Accessibility problems

### ISSUE THREAD

**Issue by timurtripp** [CuBoulder/tiamat-theme#616]
- [x] Article list preview images are missing alt text. (Fixed)
- [x] Two issues with article title background (fixed):
  - Using the overlay to improve readability of the article title does not style properly for an anonymous user, only for a logged-in user.
  - The article title still fails Firefox's contrast check even with the overlay. Adding a gradient at the bottom like the Today site uses will help to improve the readability of the article title further. Here's a comparison (D10 Express vs. D7 Express, D7 version is significantly more readable): 
<img width="1512" alt="Screenshot 2024-01-11 at 2 26 13 PM" src="https://github.com/CuBoulder/tiamat-theme/assets/22628823/20f22381-5788-4288-b42e-42a47055e389">
<img width="1512" alt="Screenshot 2024-01-11 at 2 26 00 PM" src="https://github.com/CuBoulder/tiamat-theme/assets/22628823/f7dd4193-037b-4ba0-9f9e-7ea2f8a7db48">
